### PR TITLE
Implement histogram size / content tracking for DQM PRs

### PR DESCRIPTION
**Pull request by deivydas321**

### CODE REVIEW

**Comment by dmitrijus** [comment]
hold

File: DQMServices/Components/data/html/page/chart.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<div id="main">
```
Comment (dmitrijus) [inline DQMServices/Components/data/html/page/chart.html]: Please remove this file, use https://docs.angularjs.org/api/ng/directive/script

**Comment by dmitrijus** [comment]
please test

**Comment by dmitrijus** [comment]
please test

**Comment by deivid321** [comment]
please test

**Comment by dmitrijus** [comment]
please test

**Comment by dmitrijus** [comment]
please test

**Comment by dmitrijus** [comment]
please test

**Comment by davidlange6** [comment]
for 90x (where it is also on hold for some reason)

**Pull request closed without merging**